diagnostic.js

```diff
@@ -32,25 +32,58 @@ const Movement = require('./models/movement.js');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+const create = (name, description, startYear, endYear) => {
+  Movement.create({
+      name,
+      description,
+      startYear,
+      endYear
+    })
+    .then(movement => console.log(movement.toJSON()));
+  .catch(console.error);
+};
 // Success -> console.log new Movement as JSON
 // Failure -> Console.error
 
-const index = () => {};
+const index = () => {
+  Movement.find({})
+  .then(movement => {
```

Comment:
`movements` would be a better name for this parameter since an array of documents is the return value.